README.md

```diff
@@ -347,6 +347,22 @@ To avoid storage charges for retaining the temporary composite objects, consider
 ### Google Cloud Storage Class
 Due to the quick creation and deletion of composite objects, we recommend that only the [Standard storage class](https://cloud.google.com/storage/docs/storage-classes) is applied to your bucket to minimize cost and maximize performance.
 
+### Throughput and QPS Limits
+Many machine learning efforts opt for a highly distributed training model leveraging tools such as Pytorch Lightning and Ray. These models are compatible with Dataflux, but can often trigger the rate limits of Google Cloud Storage. This typically manifest in a 429 error, or slower than expected speeds while running distributed operations. Details on specific quotas and limits can be found [here](https://cloud.google.com/storage/quotas).
+
+#### Egress Throughput Limits
+429 errors acompanied with messages indicating `This workload is drawing too much egress bandwidth from Google Cloud Storage` or `triggered the Cloud Storage Egress Bandwidth Cap` indicate that the data throughput rate of your workload is exceeding the maximum capacity of your Google Cloud Project. To address these issues, you can take the following steps:
+
+1. Check that other workloads executing within your project are not drawing excess bandwidth. Bandwidth usage can be viewed by following steps [here](https://cloud.google.com/storage/docs/bandwidth-usage#bandwidth-monitoring).
+2. Follow [these instructions](https://cloud.google.com/storage/docs/bandwidth-usage#increase) to apply for a quota increae of up to 1tbps.
+3. If more than 1Tpbs is required, contact your Technical Account Manager or Google representative to file a request on your behalf. This request should specify that you wish to increase the bandwidth caps on your project.
+4. Try [disabling](#composite-objects) object composition for Dataflux download operations. While object composition increases performance on small files, it also increases the egress bandwidth load when compared to standard download. If you are consistently hitting the bandwidth limit, disabling this feature can improve the performance of retry-backoff operations.
```

Comment:
This recommendation doesn't make sense to me. If we're hitting the throughput cap with compose enabled, then disabling compose can only lower the throughput. You may not hit the error, but that's because your throughput will be lower.

Reply:
So I'm targeting the retry of composed object downloads. It seemed from testing that the retry would max out when grabbing multiple composed downloads, but wouldn't error out when retrying on non-composed files. The issues i was encountering were surrounding any time the compose operations retried. It seems like they would have to cycle down further than normal download. Both of these errors occurred while throughput was bouncing off of the ~4.5 Tbps mark. The issue seems to occur regardless of what causes the composed object to retry. I'll just remove the suggestion for now since its more of a bug with retry, and they can always follow the suggestion under the QPS section.